examples/9.create_cancellation_signature.ts

```diff
@@ -0,0 +1,50 @@
+/**
```

Comment:
This whole example is redundant, already covered in post_cancel_order.ts so can be deleted